articles/azure/azs-how-configure-mars-backups-powershell.md

```diff
@@ -0,0 +1,91 @@
+| \$ClientID    | The application ID of a service principal with contributor permissions on Azure.                 | <form oninput="result.value=clientid.value" id="clientid" style="display: inline;"><input type="text" id="clientid" name="clientid" style="display: inline;" placeholder="00000000-0000-0000-0000-000000000000"/></form> |
+| \$ClientSecret    | A password of the service principal specified in the ClientId parameter.                 | <form oninput="result.value=clientsecret.value" id="clientsecret" style="display: inline;"><input type="text" id="clientsecret" name="clientsecret" style="display: inline;" placeholder="ftE2u]iVLs_J4+i-:q^Ltf4!&{!w3-%=3%4+}F2jk|]="/></form> |
+| \$TenantID    | The Tenant/Directory ID of your AAD domain.                 | <form oninput="result.value=tenantid.value" id="tenantid" style="display: inline;"><input type="text" id="tenantid" name="tenantid" style="display: inline;" placeholder="31537af4-6d77-4bb9-a681-d2394888ea26"/></form> |
+| \$VaultName    | The name of the recovery services vault to be created on public Azure.                 | <form oninput="result.value=vaultname.value" id="vaultname" style="display: inline;"><input type="text" id="vaultname" name="vaultname" style="display: inline;" placeholder="AzureStackVault"/></form> |
+| \$EncryptionKey    | The encryption key to encrypt the backups with.                 | <form oninput="result.value=encryptionkey.value" id="encryptionkey" style="display: inline;"><input type="text" id="encryptionkey" name="encryptionkey" style="display: inline;" placeholder="ExampleEncryptionKey"/></form> |
+| \$BackupDays    | A comma separated list of the days to backup on.                 | <form oninput="result.value=backupdays.value" id="backupdays" style="display: inline;"><input type="text" id="backupdays" name="backupdays" style="display: inline;" placeholder="Wednesday, Sunday"/></form> |
+| \$BackupTimes    | A comma separated list of the times to backup at on the backup days.                 | <form oninput="result.value=backuptimes.value" id="backuptimes" style="display: inline;"><input type="text" id="backuptimes" name="backuptimes" style="display: inline;" placeholder="16:00, 20:00"/></form> |
+| \$FoldersToBackup    | A comma separated list of folders to backup. By default backs up all drives excluding temporary storage.                 | <form oninput="result.value=folderstobackup.value" id="folderstobackup" style="display: inline;"><input type="text" id="folderstobackup" name="folderstobackup" style="display: inline;" placeholder="C:\Users, C:\Important"/></form> |
+
+1. Create a [public Azure and Azure Stack service principle name (SPN)](https://docs.ukcloud.com/articles/azure/azs-how-create-spn-powershell.html?tabs=tabid-1).
+
+2. From an elevated (administrator) PowerShell console, run the following script to download the required module and execute the backup process:
+
+<pre><code class="language-PowerShell"># Declare variables
```

Comment:
You should indent each line of code by two spaces so that it renders correctly in the numbered list.